src/print.js

```diff
@@ -23,7 +17,27 @@ function print(ast, _options) {
   );
 };
 
-module.exports = print;
+function updateBiblio(ast, _options) {
+  var options = {};
+  options.highlight = _options && _options.highlight || highlight;
+  options.biblio = _options && _options.biblio && buildBiblio(_options.biblio) || {};
+  options.head = _options && _options.head || '';
+  validateSecIDs(ast, options);
+  assignExampleNumbers(ast, options);
+  assignBiblioIDs(ast, options);
+  return options;
+}
+
+module.exports = {
+  print,
+  formatText,
+  escape,
+  escapeCode,
+  updateBiblio,
+  getTerms,
+  anchorize,
+  join
```

Comment:
These are implementation details that might change in the future - I'd suggest avoiding exporting them and directly relying on them